projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1002,6 +1002,31 @@ public function get_field_ids() {
 		return $field_ids;
 	}
 
+	/**
+	 * Get human readable error message for a given file upload error code
+	 *
+	 * @param int $error_code The error code from $_FILES['error'].
+	 * @return string Error message
+	 */
+	protected function get_upload_error_message( $error_code ) {
+		$upload_error_strings = array(
+			UPLOAD_ERR_OK         => __( 'There is no error, the file uploaded with success.', 'jetpack-forms' ),
+			UPLOAD_ERR_INI_SIZE   => __( 'The uploaded file exceeds the upload_max_filesize directive in php.ini.', 'jetpack-forms' ),
```

Comment:
Are we showing these error to users or to the admin?

Reply:
to users 😅